src/grpc.rs

```diff
@@ -223,6 +234,16 @@ pub struct ClientSettings {
     pub(crate) throw_on_append_failure: bool,
     #[serde(default)]
     pub(crate) default_user_name: Option<Credentials>,
+    #[serde(
+        default = "default_keep_alive_interval_in_ms",
```

Comment:
**SUGGESTION:** Other clients are not using `in_ms` suffixes. Might be worth align that.

Reply:
but that points to a function that returns the default `10s` value, `default` in this case is for where that default value is. It gives context to what would be a magic number instead.

Reply:
Done